### PR TITLE
issue #204185 fix: Fixing Start Event and Content Session Id Bug for Configurable Telemetry

### PR DESCRIPTION
**Pull request by ajinkyapandetekdi**

### CODE REVIEW

File: src/pages/PlayAndLearn/StartLearn.js [hunk 1]
```diff
@@ -169,13 +170,34 @@ function StartLearn() {
...
         }
       })
       .catch(err => console.log(err));
     }
 
   };
+  const [newTempContent, setNewTempContent] = useState([]); 
+
+  const handleChangeWord = () => {
+    console.log("Try New");
```
Comment (gouravmore) [inline src/pages/PlayAndLearn/StartLearn.js:184]: remove console.log

File: src/pages/PlayAndLearn/StartLearn.js [hunk 2]
```diff
@@ -169,13 +170,34 @@ function StartLearn() {
...
+    // Implement logic to select a new content
+    let getitem = content_id;
+      let old_getitem = getitem;
+      while (old_getitem === getitem) {
+        getitem = randomIntFromInterval(0, Number(newTempContent.length - 1));
+      }
+    localStorage.setItem('trysame', 'no');
```
Comment (gouravmore) [inline src/pages/PlayAndLearn/StartLearn.js:191]: What is the use of this?

File: src/pages/PlayAndLearn/StartLearn.js [hunk 3]
```diff
@@ -169,13 +170,34 @@ function StartLearn() {
...
+    localStorage.setItem('content_random_id', getitem);
```
Comment (gouravmore) [inline src/pages/PlayAndLearn/StartLearn.js:192]: are we using this anywhere?

**Review by gouravmore — CHANGES_REQUESTED** [review]

File: src/pages/ExploreAndLearn/StartLearn/StartLearn.js [hunk 4]
```diff
@@ -100,9 +98,28 @@ function StartLearn() {
...
+  const handleChangeWord = () => {
+    console.log("Try New");
+    // Implement logic to select a new content
+    let getitem = content_id;
+      let old_getitem = getitem;
+      while (old_getitem === getitem) {
+        getitem = randomIntFromInterval(0, Number(newTempContent.length - 1));
+      }
+    localStorage.setItem('trysame', 'no');
+    localStorage.setItem('content_random_id', getitem);
```
[the remark this answers is not in the document]
Comment (ajinkyapandetekdi) [reply src/pages/ExploreAndLearn/StartLearn/StartLearn.js:111]: Yes we are using it for S3 bucket to get audioFile name in VoiceCompare.js.

File: src/pages/ExploreAndLearn/StartLearn/StartLearn.js [hunk 5]
```diff
@@ -100,9 +98,28 @@ function StartLearn() {
...
+
+  const handleChangeWord = () => {
+    console.log("Try New");
+    // Implement logic to select a new content
+    let getitem = content_id;
+      let old_getitem = getitem;
+      while (old_getitem === getitem) {
+        getitem = randomIntFromInterval(0, Number(newTempContent.length - 1));
+      }
+    localStorage.setItem('trysame', 'no');
```
[the remark this answers is not in the document]
Comment (ajinkyapandetekdi) [reply src/pages/ExploreAndLearn/StartLearn/StartLearn.js:110]: To stop reloading of page i am using virtualDom (useState).

File: src/pages/PlayAndLearn/StartLearn.js (hunk 1 above)
Comment (ajinkyapandetekdi) [reply src/pages/PlayAndLearn/StartLearn.js:184]: removed

File: src/pages/PlayAndLearn/StartLearn.js (hunk 2 above)
Comment (ajinkyapandetekdi) [reply src/pages/PlayAndLearn/StartLearn.js:191]: to get random Word or sentence

File: src/pages/PlayAndLearn/StartLearn.js (hunk 3 above)
Comment (ajinkyapandetekdi) [reply src/pages/PlayAndLearn/StartLearn.js:192]: yes we are using it in VoiceComapre.js line no 219

File: src/pages/ExploreAndLearn/StartLearn/StartLearn.js [hunk 6]
```diff
@@ -100,9 +98,28 @@ function StartLearn() {
   const [hide_navFooter, set_hide_navFooter] = useState('false');
   const [load_cnt, set_load_cnt] = useState(0);
   // const [content_list, setContent_list] = useState(null);
+  const [newTempContent, setNewTempContent] = useState([]); 
+
+  const handleChangeWord = () => {
+    console.log("Try New");
```
[the remark this answers is not in the document]
Comment (ajinkyapandetekdi) [reply src/pages/ExploreAndLearn/StartLearn/StartLearn.js]: removed it

**Pull request merged**